### ISSUE TITLE
add test for example

### ISSUE THREAD

**Issue by CJ-Wright** [conda-forge/staged-recipes#1820]
Add something so that the CI fails if the example folder, and meta.yml are missing.

**Comment by jakirkham** [comment]
This should be pretty easy. Adding something like `test -d` and `test -f` on Travis CI and/or CircleCI should do the trick. Could throw in an [`if not exist`]( https://github.com/conda-forge/boost-feedstock/blob/9352548781364ed86c4c6ab750725097ff44fc61/recipe/meta.yaml#L41 ) for AppVeyor too. Would you like to do a PR, @CJ-Wright?

**Comment by CJ-Wright** [comment]
I could potentially put together a PR, although not this week most likely.

**Comment by jakirkham** [comment]
Are you still interested in this, @CJ-Wright?

**Comment by CJ-Wright** [comment]
Yes but it would most likey be better if someone took it off my hands since I am overloaded as it is
